whois-commons/src/test/java/net/ripe/db/whois/common/hazelcast/TestHazelcastBlockedIps.java

```diff
@@ -0,0 +1,49 @@
+package net.ripe.db.whois.common.hazelcast;
+
+import com.google.common.base.Joiner;
+import net.ripe.db.whois.common.ip.IpInterval;
+import net.ripe.db.whois.common.profiles.WhoisProfile;
+import org.mockito.internal.util.collections.Sets;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+
+import java.util.Set;
+
+@Component
+@Profile({WhoisProfile.TEST})
+public class TestHazelcastBlockedIps implements HazelcastBlockedIps {
```

Comment:
rename to TestBlockedIps?

Reply:
fair enough because we are not using hazelcast there. OK

Reply:
Sorry if it was not clear. I meant to say the interface name should not be hazelcast as you have two implementation one with hazelcast and one without so service layer should have hazelcast as prefix not interface.

Reply:
I changed the name of the interface